### DISCUSSION TITLE
Welcome to Gisola Discussions!

### DISCUSSION THREAD

**Discussion by nikosT** [nikosT/Gisola#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Arjunvh** [comment]
is it ready for installation and testing??

**Reply by nikosT** [reply]
Not yet. However, If no further delays occurred, it will be released in mid-August. I suggest to you to subscribe to the repo in order to be notified about the release.

**Comment by Donavin97** [comment]
Hi, I have a question regarding running Gisola.
This version is optimized for multicore machines if I am correct?
Would it work on a midrange conventional PC as well?
I suppose it would just take a bit more time for MT calculation, as apposed to running on a compute cluster?
Thanks.

**Reply by nikosT** [reply]
Hi @Donavin97 , 
Exactly as you said, apart from that the program currently supports single node (1 PC/server) with multicore/manycore capabilities. The configuration is also adjustable, so you can have a dense grid, thus more computations or a sparse one for less computations.

**Comment by Donavin97** [comment]
Thanks @nikosT. Now everything makes sence.
I am already preparing a test dataset covering light to moderate earthquakes in Southern Africa to test Gisola with, once it's released.
Keep up the good work, and once again thanks for sharing such a useful project with us.

**Comment by nikosT** [comment]
@Donavin97 and all, as you probably already saw the first release of Gisola just came out!

[https://doi.org/10.1785/0220210153](https://doi.org/10.1785/0220210153)

**Reply by Donavin97** [reply]
Thanks @nikosT  for this awesome project. It already installed and running on my system. I have computed 3 mt solutions in realtime so far. May I fork the repository and add a few things to it?

**Reply by nikosT** [reply]
Sure @Donavin97 , you don't need to ask for it!
You could also create a Pull Request (PR) to Gisola, if you think that it could enhance the software with more features.
